module-03-react/src/App.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useRef, useState } from "react";
+import "./App.css";
+import Gallery from "./components/Gallery";
+import SideMenu from "./components/SideMenu";
+import TopMenu from "./components/TopMenu";
+import BottomBtn from "./components/BottomBtn";
+import Modal from "./components/Modal";
+
+
+function App() {
+  const galleryRef = useRef();
+  const [url, setUrl] = useState("people/");
```

Comment:
This approach of using a simple string state for URL routing is limited and doesn't leverage browser history functionality. Consider implementing React Router (react-router-dom) instead, which would provide proper URL-based navigation, browser history integration, and more robust routing capabilities. React Router would allow for bookmarkable URLs, back/forward navigation support, and a more maintainable routing structure as the application grows

https://reactrouter.com/